spec/features/editing_content/insert_contact_no_js_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.feature "Insert contact without Javascript" do
+  include AccessibleAutocompleteHelper
+
+  scenario do
+    given_there_is_an_edition
+    when_i_go_to_edit_the_edition
+    and_i_go_to_add_a_contact
+    when_i_select_a_contact
+    then_i_can_see_the_contact_in_the_body
```

Comment:
I think we need to change the behaviour here, as we'll be trying to insert the snippet into a potentially out-of-date version of the body field - we're no longer saving the page and redirecting.

Reply:
Thank for the review Ben! I updated as discussed. Do let me know if something feels shady, especially on the Rails side